discovery-provider/src/tasks/entity_manager/playlist.py

```diff
@@ -42,7 +42,7 @@ def validate_playlist_tx(params: ManageEntityParameters):
 
     if params.action == Action.CREATE:
         if playlist_id in params.existing_records[EntityType.PLAYLIST]:
-            raise Exception(f"Cannot create playlist {playlist_id} that already exists")
+            raise Exception(f"Cannot create playlist {playlist_id} that already exist")
```

Comment:
i think this one is actually right as is

Reply:
lol true